packages/flutter/lib/src/material/icons.dart

```diff
@@ -0,0 +1,945 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+class IconData {
```

Comment:
Maybe rename this to IconGlyph? IconData made me think it might include e.g. color and bidi information (for those icons that reflect in rtl situations).